pallets/ethereum-events/src/lib.rs

```diff
@@ -738,21 +739,11 @@ pub mod pallet {
         #[pallet::weight(
             <T as pallet::Config>::WeightInfo::set_ethereum_contract_map_storage().max(<T as Config>::WeightInfo::set_ethereum_contract_storage()
         ))]
-        pub fn set_ethereum_contract(
-            origin: OriginFor<T>,
-            contract_type: EthereumContracts,
-            contract_address: H160,
-        ) -> DispatchResult {
+        pub fn map_nft_contract(origin: OriginFor<T>, contract_address: H160) -> DispatchResult {
```

Comment:
not sure if the function name is describing the action. If all its doing is insert nft contract address then why not call it that?